src/main/resources/production.yml

```
@@ -0,0 +1,5 @@
+server: localhost:13306
+database: chess
+option: ?useSSL=false&allowPublicKeyRetrieval=true&serverTimezone=UTC
+username: root
+password: root
```

```suggestion
db: 
  server: localhost:13306
  database: chess
  option: ?useSSL=false&allowPublicKeyRetrieval=true&serverTimezone=UTC
  username: root
  password: root
```
db 혹은 database로 어떤 것에대한 값들인지를 계층을 명시하면 더 좋을것 같아요!
나중에 다른 properties 들이 들어올 수 있으니까요